core/App/ehsm_napi.cpp

```diff
@@ -81,11 +81,11 @@ void NAPI_Finalize(){
 */
 char* NAPI_CreateKey(const uint32_t keyspec, const uint32_t origin)
 {
+    RetJsonObj retJsonObj;
     ehsm_status_t ret = EH_OK;
-    ehsm_keyblob_t master_key;
+    ehsm_keyblob_t master_key = {0};
 
```

Comment:
you can add an open here, not sure if it worked for the nested struture initialization.

it's more better to call the memset.

Reply:
It will be change.